openverse_catalog/dags/util/loader/columns.py

```diff
@@ -0,0 +1,302 @@
+    ),
+    col.LAST_SYNCED: DbColumn(
+        "last_synced_with_source", Datatype.timestamp, None, UpsertStrategy.now
+    ),
+    col.REMOVED: DbColumn(
+        "removed_from_source", Datatype.char, "varying(3000)", UpsertStrategy.false
+    ),
+    col.DURATION: DbColumn("duration", Datatype.int),
+    col.BIT_RATE: DbColumn("bit_rate", Datatype.int),
+    col.SAMPLE_RATE: DbColumn("sample_rate", Datatype.int),
+    col.CATEGORY: DbColumn("category", Datatype.char, "varying(100)"),
+    col.GENRES: DbColumn(
+        "genres", Datatype.char, "varying(80)[]", UpsertStrategy.merge_array
+    ),
+    col.AUDIO_SET: DbColumn(
+        "audio_set", Datatype.jsonb, None, UpsertStrategy.merge_jsonb_objects
+    ),
+    col.ALT_FILES: DbColumn(
+        "alt_files", Datatype.jsonb, None, UpsertStrategy.merge_jsonb_objects
+    ),
+}
+
+# Dictionary of lists of column names common for all media type,
+# for the intermediary loading table, and the main `image` or `audio` table
+# in the Upstream (also called Openledger db)
+common_columns = {
+    "loading": [
+        col.FOREIGN_ID,
+        col.LANDING_URL,
+        col.DIRECT_URL,
+        col.THUMBNAIL,
+        col.FILESIZE,
+        col.LICENSE,
+        col.LICENSE_VERSION,
+        col.CREATOR,
+        col.CREATOR_URL,
+        col.TITLE,
+        col.META_DATA,
+        col.TAGS,
+        col.WATERMARKED,
+        col.PROVIDER,
+        col.SOURCE,
+        col.INGESTION_TYPE,
+    ],
+    "main": [
+        # IDENTIFIER,
+        col.CREATED_ON,
+        col.UPDATED_ON,
+        col.INGESTION_TYPE,
+        col.PROVIDER,
+        col.SOURCE,
+        col.FOREIGN_ID,
+        col.LANDING_URL,
+        col.DIRECT_URL,
+        col.THUMBNAIL,
+        col.FILESIZE,
+        col.LICENSE,
+        col.LICENSE_VERSION,
+        col.CREATOR,
+        col.CREATOR_URL,
+        col.TITLE,
+        col.LAST_SYNCED,
+        col.REMOVED,
+        col.META_DATA,
+        col.TAGS,
+        col.WATERMARKED,
+        # LAST_SYNCED
+        # REMOVED
+    ],
+}
+
+# Dictionary of lists of column names that are specific to the media type.
+# These columns are at the end of the db table.
+media_columns = {
+    AUDIO: [
+        col.DURATION,
+        col.BIT_RATE,
+        col.SAMPLE_RATE,
+        col.CATEGORY,
+        col.GENRES,
+        col.AUDIO_SET,
+        col.ALT_FILES,
+    ],
+    IMAGE: [col.WIDTH, col.HEIGHT],
+}
+
+
+def get_table_columns(media_type: str, table_type: str = "main") -> List[str]:
+    """
+    Returns the list of column names that a loading or main db table
+    for the media type should have.
+    :param media_type: One of the supported media types, from
+    `util/constants.py`
+    :param table_type: Whether it is intermediary loading table, or the main
+    `image` or `audio` table.
+    :return: list of strings - names of the columns for selected db table
+    """
+    if table_type not in ["main", "loading"]:
```

Comment:
I know that the set look up is faster, but since there are only two members in the list, I thought it wouldn't make much difference (or would it?), and so used the more familiar list :)
Consistency is best, anyways, so thank you for this suggestion!

Reply:
Totally, it doesn't much matter here haha. I like having it as a pattern though because it's one of those places that could bring unexpected speed-ups!